README.md

```diff
@@ -224,6 +225,166 @@ There are number of techniques you can use to ensure that access token expiry is
+                                                                 block:^(TVOConnectOptionsBuilder *builder) {
+    builder.region = region;
+}];
+
+TVOAcceptOptions *options = [TVOAcceptOptions optionsWithCallInvite:callInvite
+                                                              block:^(TVOAcceptOptionsBuilder *builder) {
+    builder.region = region;
+}];
+```
+
+### TVOConnectOptions & TVOAcceptOptions
+To support configurability upon making or accepting a call, new classes have been added. Create a `TVOConnectOptions` object and make configurations via the `TVOConnectOptionsBuilder` in the `block`. Once `TVOConnectOptions` is created it can be provided when connecting a Call as shown below:
+
+```.objc
+TVOConnectOptions *options = [TVOConnectOptions optionsWithAccessToken:accessToken
+                                                                 block:^(TVOConnectOptionsBuilder *builder) {
+    builder.params = params;
+}];
+
+self.call = [TwilioVoice connectWithOptions:options delegate:self];
+```
+
+A `TVOCallInvite` can also be accepted using `TVOAcceptOptions` as shown below:
+
+```.objc
+TVOAcceptOptions *options = [TVOAcceptOptions optionsWithCallInvite:callInvite
+                                                              block:^(TVOAcceptOptionsBuilder *builder) {
+    builder.region = region;
+}];
+
+self.call = [callInvite acceptWithOptions:options delegate:self];
+```
+
+### Ringing
+Ringing is now provided as a call state. The delegate method `callDidStartRinging:` corresponding to this state transition is called once before the `callDidConnect:` method when the callee is being alerted of a Call. The behavior of this callback is determined by the `answerOnBridge` flag provided in the `Dial` verb of your TwiML application associated with this client. If the `answerOnBridge` flag is `false`, which is the default, the `callDidConnect:` callback will be called immediately after `callDidStartRinging:`. If the `answerOnBridge` flag is `true`, this will cause the `callDidConnect:` method being called only after the Call is answered. See [answerOnBridge](https://www.twilio.com/docs/voice/twiml/dial#answeronbridge) for more details on how to use it with the `Dial` TwiML verb. If the TwiML response contains a `Say` verb, then the `callDidConnect:` method will be called immediately after `callDidStartRinging:` is called, irrespective of the value of `answerOnBridge` being set to `true` or `false`.
+
+These changes are added as follows:
+
+```.objc
+// TVOCall.h
+typedef NS_ENUM(NSUInteger, TVOCallState) {
+    TVOCallStateConnecting = 0, ///< The Call is connecting.
+    TVOCallStateRinging,        ///< The Call is ringing.
+    TVOCallStateConnected,      ///< The Call is connected.
+    TVOCallStateDisconnected    ///< The Call is disconnected.
+};
+
+// TVOCallDelegate.h
+@protocol TVOCallDelegate
+
+@optional
+- (void)callDidStartRinging:(nonnull TVOCall *)call;
+
+@end
+```
+
```

Comment:
Please add a section for https://github.com/twilio/voice-quickstart-android/tree/3.x#media-establishment--connectivity